### PR TITLE
Add CC Attribution

### PR DESCRIPTION
**Pull request by pzalep1**
Screen Shot 2019-04-26 at 8 59 43 AM

One with a longer title. The copy button is aligned with the top line of text.
Screen Shot 2019-04-26 at 9 02 32 AM

Screen Shot 2019-04-25 at 4 10 33 PM

### CODE REVIEW

File: src/app/cube/details/action-panel/action-panel.component.ts [hunk 1]
```diff
@@ -143,12 +144,24 @@ export class ActionPanelComponent implements OnInit, OnDestroy {
 
   copyLink() {
     const el = this.objectLinkElement.nativeElement;
+    console.log(el);
```
Comment (DonSeannelly) [inline src/app/cube/details/action-panel/action-panel.component.ts]: Looks like a log made its way in here

**Comment by siddharthkaza** [comment]
Change text to “Attribute this Object”

**Comment by pzalep1** [comment]
> Change text to “Attribute this Object”

Resolved with Change text to Attribute this Object

File: src/app/cube/details/action-panel/action-panel.component.ts (hunk 1 above)
Comment (pzalep1) [reply src/app/cube/details/action-panel/action-panel.component.ts]: Resolved with Remove console log

File: src/app/cube/details/action-panel/action-panel.component.ts [hunk 2]
```diff
@@ -149,6 +150,17 @@ export class ActionPanelComponent implements OnInit, OnDestroy {
...
 
+  copyAttribution() {
+    const range = document.createRange();
+    range.selectNode(document.getElementById('objectAttribution'));
+    window.getSelection().removeAllRanges();
+    window.getSelection().addRange(range);
+    document.execCommand('copy');
+    window.getSelection().removeAllRanges();
+
+    this.toaster.notify('Success!', 'Learning object Creative Commons Attribution copied to your clipboard!', 'good', 'far fa-check');
```
Comment (DonSeannelly) [inline src/app/cube/details/action-panel/action-panel.component.ts]: Learning Object should be capitalized:
```suggestion
    this.toaster.notify('Success!', 'Learning Object Creative Commons Attribution copied to your clipboard!', 'good', 'far fa-check');
```

But maybe we can simplify this a little more.
```suggestion
    this.toaster.notify('Success!', 'Attribution information has been copied to your clipboard!', 'good', 'far fa-check');
```

File: src/app/cube/details/action-panel/action-panel.component.ts [hunk 3]
```diff
@@ -149,6 +150,17 @@ export class ActionPanelComponent implements OnInit, OnDestroy {
     this.toaster.notify('Success!', 'Learning object link copied to your clipboard!', 'good', 'far fa-check');
   }
 
+  copyAttribution() {
```
Comment (DonSeannelly) [inline src/app/cube/details/action-panel/action-panel.component.ts]: This should have JSDoc

**Pull request merged**